tests/broker-engine/opentelemetry.robot

```diff
@@ -116,30 +116,31 @@ BEOTEL_TELEGRAF_CHECK_HOST
 
     ${resources_list}    Ctn Create Otl Request    ${0}    host_1
 
+    # check without feed
+    ${start}    Ctn Get Round Current Date
+    Ctn Schedule Forced Host Check    host_1
+    ${result}    Ctn Check Host Check Status With Timeout
```

Comment:
Since you are in BBDO3, I think it is better to check the resources table, the keyword is something like "Ctn Check Resources Host Check Status With Timeout"...

Reply:
done